site/source/docs/tools_reference/emcc.rst

```diff
@@ -202,6 +202,8 @@ Options that are modified or new in *emcc* are listed below:
 
 	You normally don't need to specify this option, as ``-O`` with an optimization level will set a good value.
 
+	.. note:: Some options might override this flag (e.g. ``EMTERPRETIFY``, ``DEAD_FUNCTIONS``, ``OUTLINING_LIMIT``, ``SAFE_HEAP`` and ``SPLIT_MEMORY`` override the value with ``js-opts=1``).
```

Comment:
also here, please mention that they do so because they depend on the js-optimizer.